weave

```diff
@@ -801,6 +857,7 @@ case "$COMMAND" in
     expose)
         collect_cidr_args "$@"
         shift $CIDR_COUNT
+        [ $CIDR_COUNT -gt 0 ] || { get_ipam_cidr weave && expose $CIDR_ARGS ; }
```

Comment:
Turns out it doesn't matter what you told a weave which has been restarted.  For correct operation, populate_ipam() should include the bridge address.